### PR TITLE
data card for Zgamma

### PR DESCRIPTION
**Pull request by doanhien**
data card and run card for aMCatNLO
run card for sherpa

### CODE REVIEW

**Comment by efeyazgan** [comment]
@GurpreetSinghChahal could you please take a look? thanks.

File: mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   F  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
```
Comment (efeyazgan) [inline mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: Could you please set this to True? thanks.

File: mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  263000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (efeyazgan) [inline mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: Do you have the same pdf in the sherpa case?

**Comment by efeyazgan** [comment]
This says fxfx but you have mlm in the card (1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching) - please correct.

File: mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat (hunk 1 above)
Comment (doanhien) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]:
```suggestion
   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
```

Comment (doanhien) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: Ok. It is set to True.

File: mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat (hunk 2 above)
Comment (doanhien) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]:
```suggestion
  263000    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number 
```

Comment (doanhien) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: No. I use default PDF for sherpa which is CT10.

**Comment by doanhien** [comment]
Hi, 
It must be MLM. I don't know how to change the file name from here.
Could you please tell me how to do that?

File: mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat (hunk 2 above)
Comment (efeyazgan) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: I thought Sherpa default is NNPDF30NNLO. @GurpreetSinghChahal ?

Comment (doanhien) [reply mg_v26x/ZATo2LA0123j_5f_LO_FXFX/ZATo2LA0123j_5f_LO_FXFX_run_card.dat]: I read sherpa manual standalone, the default is CT10. 
And I just checked the log file when running sherpack, the default is NNPDF30NNLO.
You are right.

**Comment by menglu21** [comment]
> > > Hi,
> > > It must be MLM. I don't know how to change the file name from here.
> > > Could you please tell me how to do that?
> > 
> > 
> > Hi Hien, i once thought you want to produce FxFx sample, so if you just want to have MLM samples with up to 3jets, then we already have SMP-RunIISummer15wmLHEGS-00267 in production, and the gridpack is here /cvmfs/cms.cern.ch/phys_generator/gridpacks/slc6_amd64_gcc481/13TeV/madgraph/V5_2.4.2/ZGToLLG_0123j_SM_5f_LO_MLM/ZGToLLG_madgraphMLM_slc6_amd64_gcc481_CMSSW_7_1_30_tarball.tar.xz
> 
> Hi Meng,
> Is this gridpack produced with magraph V2.4.2 or V2.6?
> If I remember correctly, there was problem with V2.4, so we have to use V2.6 to produce.
> Can you please confirm?
> Thanks.

Hi Hien,
the previous issue of V2.4 only have effect on FxFx samples containing photon, it's fine for LO samples.

File: mg_v26x/ZATo2LA0123j_5f_LO_MLM/ZATo2LA0123j_5f_LO_MLM_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+define j = j b b~
+
+define lep = e+ mu+ ta+ e- mu- ta-
+
+generate p p > lep lep a @0
+add process p p > lep lep j a @1
+add process p p > lep lep j j a @2
+add process p p > lep lep j j j a @3
+
+output ZATo2LA0123j_5f_LO_FXFX -nojpeg
```
Comment (efeyazgan) [inline mg_v26x/ZATo2LA0123j_5f_LO_MLM/ZATo2LA0123j_5f_LO_MLM_proc_card.dat]: just for consistency can you please change FXFX to MLM

**Comment by qliphy** [comment]
@doanhien  The cards are currently in a strange place as mg_v26x/ZATo2LA0123j_5f_LO_MLM

 Please make a new PR and commit to a right directory under genproductions/bin/MadGraph5_aMCatNLO/cards/production/

**Pull request merged**